### PR TITLE
add a small doc to show how to use sentry in Fargate service in ca go

### PR DESCRIPTION
**Pull request by Allen-Ning**
add a small doc to show how to use sentry in Fargate service in ca go

### Context
Ticket: https://cultureamp.atlassian.net/browse/COSE-527

### CODE REVIEW

**Review by joshjon — APPROVED** [review]
LGTM

**Comment by El-Wel** [comment]
The documentation looks good but using a readme file isn’t consistent with the existing documentation in this repo, might I suggest adding this to the existing documentation in this directory?

File: x/sentry/errorreport/examples_test.go [hunk 1]
```diff
@@ -94,3 +99,55 @@ func processRecord(ctx context.Context, record events.KinesisEventRecord) error
 	// do something with the record
 	return nil
 }
+
+func Example_second() {
+	// This is an example of how to use the errorreport package in a Main
```
Comment (El-Wel) [inline x/sentry/errorreport/examples_test.go:104]: The above example is also of a main function but specifies it is for a lambda, is this example of a fargate main?

File: x/sentry/errorreport/examples_test.go [hunk 2]
```diff
@@ -94,3 +99,55 @@ func processRecord(ctx context.Context, record events.KinesisEventRecord) error
 	// do something with the record
 	return nil
 }
+
+func Example_second() {
```
Comment (El-Wel) [inline x/sentry/errorreport/examples_test.go:103]: The name of this function will show on go doc as Example (Second) (attached screenshot), could these two examples be named to describe what they are doing like Example (Lambda) and Example (Fargate)? Or are they both just examples of the same thing?
Screen Shot 2022-09-19 at 2 32 14 pm

Comment (Allen-Ning) [reply x/sentry/errorreport/examples_test.go:103]: > The name of this function will show on go doc as Example (Second) (attached screenshot), could these two examples be named to describe what they are doing like Example (Lambda) and Example (Fargate)? Or are they both just examples of the same thing? Screen Shot 2022-09-19 at 2 32 14 pm

Unfortunately, you can only name them `example`, `example_second`, `example_third` for package level examples.
E.g. `example_for_lambda` will not work

Comment (El-Wel) [reply x/sentry/errorreport/examples_test.go:103]: Ah dang, I was able to use it locally. Is this feature just not supported on pkg.go.dev?
Screen Shot 2022-09-19 at 2 40 20 pm

Comment (El-Wel) [reply x/sentry/errorreport/examples_test.go:103]: It seems that `Example_for_fargate` does not work but `Example_fargate` does. Looks like it just can't support more than one additional word

Comment (Allen-Ning) [reply x/sentry/errorreport/examples_test.go:103]: > It seems that `Example_for_fargate` does not work but `Example_fargate` does. Looks like it just can't support more than one descriptive word

Yes. The naming convention is very limited. Happy to change it to `Example_xxx` if it makes more sense for u.
Screen Shot 2022-09-19 at 2 48 16 pm

Comment (El-Wel) [reply x/sentry/errorreport/examples_test.go]: I think having a descriptive suffix is more helpful, especially as this is a public repository

**Review by El-Wel — APPROVED** [review]
Thanks for the discussion and updates! Looks great!

**Pull request merged**